test.js

```diff
@@ -325,3 +325,170 @@ ${dimSide}foo${dimSide}
 ${dimBottomBorder}
 	`);
 });
+
+test('no wrapping when content = columns - 2 and no padding and no margin', t => {
+	const width = process.stdout.columns;
+	const longContent = 'x'.repeat(width - 2);
+	const box = boxen(longContent);
+
+	// No endlines
+	t.is(box.indexOf('\n'), -1);
```

Comment:
```suggestion
	t.false(box.includes('\n'));
```

Same with the other places this is done.